.github/workflows/e2e.yml

```diff
@@ -35,11 +35,14 @@ jobs:
 
     - name: Install Docci Readme Runner
       run: |
-        VERSION=v0.9.0-alpha.2
+        VERSION=v0.9.0-beta.2
         wget -O docci.tar.gz "https://github.com/Reecepbcups/docci/releases/download/${VERSION}/docci_Linux_x86_64.tar.gz"
         tar -xzf docci.tar.gz
         sudo mv docci /usr/local/bin/docci
         sudo chmod +x /usr/local/bin/docci
         rm docci.tar.gz
 
-    - run: docci run DEV.md --cleanup-commands="killall anvil" --hide-background-logs
+    # TODO: build and cache docker image of wavs-middleware here for future runs?
+    - run: docci run DEV.md --cleanup-commands="killall anvil || true" --hide-background-logs
+    - run: docci run DEV_1.md --cleanup-commands="killall anvil || true" --hide-background-logs
+    - run: docci run DEV_2.md --cleanup-commands="killall anvil || true" --hide-background-logs
```

Comment:
I do not know what you want the filenames to be, so I can change those. lmk (same for the title within the markdown file, left as TODO for now)